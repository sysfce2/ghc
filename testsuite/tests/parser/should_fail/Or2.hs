{-# LANGUAGE OrPatterns #-}

module Main where

main = print $ case 3 of
  4 | 5 -> False