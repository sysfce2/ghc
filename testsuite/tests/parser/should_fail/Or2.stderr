
Or2.hs:6:7: error: [GHC-39999]
    • No instance for ‘Num Bool’ arising from the literal ‘5’
    • In the expression: 5
      In a stmt of a pattern guard for
                     a case alternative:
        5
      In a case alternative: 4 | 5 -> False
